src/Metadata/MetadataMatcher.php

```diff
@@ -35,12 +35,6 @@ public function withMetadataMatch(
             $fieldType = FieldType::METADATA();
         }
 
-        if ($fieldType->is(FieldType::MESSAGE_PROPERTY())
-            && ! in_array($field, ['uuid', 'createdAt', 'created_at', 'messageName', 'message_name'], true)
-        ) {
-            throw new InvalidArgumentException(sprintf('Invalid message property "%s" given', $field));
```

Comment:
but pdo-event-store can map them. I'd like to provide default filters with the event-store-mgmt-ui and I don't know which event-store implementation is used.